ttnn/cpp/ttnn/operations/normalization/layernorm/device/kernels/dataflow/writer_unary_sharded_ln_rm_gb.cpp

```diff
@@ -67,8 +67,9 @@ void kernel_main() {
         for (uint32_t w = 0; w < block_w; w++) {
             uint32_t tile_id = gamma_tile_start_id + w;
             uint64_t gamma_noc_addr = get_noc_addr(tile_id, gamma);
-            noc_async_read(gamma_noc_addr, l1_write_addr_gamma, mask_read_tile_face_bytes);
-            gamma_noc_addr += mask_read_tile_face_bytes;
+            noc_async_read(gamma_noc_addr, l1_write_addr_gamma, mask_read_tile_face_bytes * 2);
```

Comment:
Minor: Could you make the declaration of `mask_read_tile_face_bytes` and `mask_read_tile_offset_bytes` constexpr?